src/test/java/com/znsio/rpap/AppBaseTest.java

```diff
@@ -0,0 +1,58 @@
+package com.znsio.rpap;
+
+import com.znsio.applitools.integration.ApplitoolsInitializer;
+import com.znsio.rpap.pages.Page;
+import com.znsio.rpap.utils.DriverFactory;
+import io.appium.java_client.AppiumDriver;
+import com.znsio.reportportal.integration.properties.Config;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+import java.net.MalformedURLException;
+import java.time.Duration;
+import java.util.Properties;
+import static com.znsio.reportportal.integration.utils.ReportPortalLogger.logInfoMessage;
+
+
+public class AppBaseTest extends ApplitoolsInitializer {
+
+    protected static AppiumDriver appDriver;
+    private static WebDriverWait wait;
+    private static final Logger LOGGER = Logger.getLogger(AppBaseTest.class.getName());
+    private static final Properties config = Config.loadProperties(System.getProperty("CONFIG"));
+    public static final String PAGE_LOAD_TIME = config.getProperty("PAGE_LOAD_TIME");
+    private static final String APP_ACTIVITY = config.getProperty("APP_ACTIVITY");
+    private static final String APP_PACKAGE_NAME = config.getProperty("APP_PACKAGE_NAME");
+    private static final String AUTOMATION_NAME = config.getProperty("AUTOMATION_NAME");
+    private static final String PLATFORM = config.getProperty("PLATFORM");
```

Comment:
Yes, they can be used, but in the method

        appDriver = DriverFactory.launchMobileApp(config.getProperty("PLATFORM"),config.getProperty("AUTOMATION_NAME"), config.getProperty("APP_PACKAGE_NAME"), config.getProperty("APP_ACTIVITY"), APK_LOCATION);
        
        when we are writing in this manner - then it is getting exceed to second line, and out of visible scope
        So for sake of readibility ,have written in this way

Reply:
I'm not asking to use it directly in the method name, but you can declare these variables as local variables in the method if we're only using them in one method instead of create them as data members of the class

Reply:
Moved them